Relocation overflow errors when linking on magic leap

This is tracking the fact that the Magic Leap toolchain is using an old version of binutils. Certain PRs push the linker over the edge like this:
```
[Servo2D] Linking program Servo2D...
/Users/worker/tasks/task_1558764312/magicleap/v0.17.0/lumin/stl/libc++/lib/libc++abi.a(cxa_demangle.o): In function `std::__ndk1::char_traits<char>::copy(char*, char const*, unsigned long)':
/Volumes/android/aosp_sdk_osx/repos/external/libcxx/../../external/libcxxabi/../libcxx/include/string:655:(.text._ZN10__cxxabiv112_GLOBAL__N_118parse_substitutionINS0_2DbEEEPKcS4_S4_RT_+0x10dc): relocation truncated to fit: R_AARCH64_CALL26 against symbol `memcpy@@LIBC' defined in .text section in /Users/worker/tasks/task_1558764312/magicleap/v0.17.0/lumin/usr/lib/libc.so
/Volumes/android/aosp_sdk_osx/repos/external/libcxx/../../external/libcxxabi/../libcxx/include/string:655:(.text._ZN10__cxxabiv112_GLOBAL__N_118parse_substitutionINS0_2DbEEEPKcS4_S4_RT_+0x1158): relocation truncated to fit: R_AARCH64_CALL26 against symbol `memcpy@@LIBC' defined in .text section in /Users/worker/tasks/task_1558764312/magicleap/v0.17.0/lumin/usr/lib/libc.so
/Volumes/android/aosp_sdk_osx/repos/external/libcxx/../../external/libcxxabi/../libcxx/include/string:655:(.text._ZN10__cxxabiv112_GLOBAL__N_118parse_substitutionINS0_2DbEEEPKcS4_S4_RT_+0x1204): relocation truncated to fit: R_AARCH64_CALL26 against symbol `memcpy@@LIBC' defined in .text section in /Users/worker/tasks/task_1558764312/magicleap/v0.17.0/lumin/usr/lib/libc.so
/Volumes/android/aosp_sdk_osx/repos/external/libcxx/../../external/libcxxabi/../libcxx/include/string:655:(.text._ZN10__cxxabiv112_GLOBAL__N_118parse_substitutionINS0_2DbEEEPKcS4_S4_RT_+0x1290): relocation truncated to fit: R_AARCH64_CALL26 against symbol `memcpy@@LIBC' defined in .text section in /Users/worker/tasks/task_1558764312/magicleap/v0.17.0/lumin/usr/lib/libc.so
/Volumes/android/aosp_sdk_osx/repos/external/libcxx/../../external/libcxxabi/../libcxx/include/string:655:(.text._ZN10__cxxabiv112_GLOBAL__N_120parse_template_paramINS0_2DbEEEPKcS4_S4_RT_+0x198): relocation truncated to fit: R_AARCH64_CALL26 against symbol `memcpy@@LIBC' defined in .text section in /Users/worker/tasks/task_1558764312/magicleap/v0.17.0/lumin/usr/lib/libc.so
/Volumes/android/aosp_sdk_osx/repos/external/libcxx/../../external/libcxxabi/../libcxx/include/string:655:(.text._ZN10__cxxabiv112_GLOBAL__N_120parse_template_paramINS0_2DbEEEPKcS4_S4_RT_+0x214): relocation truncated to fit: R_AARCH64_CALL26 against symbol `memcpy@@LIBC' defined in .text section in /Users/worker/tasks/task_1558764312/magicleap/v0.17.0/lumin/usr/lib/libc.so
/Volumes/android/aosp_sdk_osx/repos/external/libcxx/../../external/libcxxabi/../libcxx/include/string:655:(.text._ZN10__cxxabiv112_GLOBAL__N_120parse_template_paramINS0_2DbEEEPKcS4_S4_RT_+0x2c0): relocation truncated to fit: R_AARCH64_CALL26 against symbol `memcpy@@LIBC' defined in .text section in /Users/worker/tasks/task_1558764312/magicleap/v0.17.0/lumin/usr/lib/libc.so
/Volumes/android/aosp_sdk_osx/repos/external/libcxx/../../external/libcxxabi/../libcxx/include/string:655:(.text._ZN10__cxxabiv112_GLOBAL__N_120parse_template_paramINS0_2DbEEEPKcS4_S4_RT_+0x348): relocation truncated to fit: R_AARCH64_CALL26 against symbol `memcpy@@LIBC' defined in .text section in /Users/worker/tasks/task_1558764312/magicleap/v0.17.0/lumin/usr/lib/libc.so
/Volumes/android/aosp_sdk_osx/repos/external/libcxx/../../external/libcxxabi/../libcxx/include/string:655:(.text._ZN10__cxxabiv112_GLOBAL__N_120parse_template_paramINS0_2DbEEEPKcS4_S4_RT_+0x634): relocation truncated to fit: R_AARCH64_CALL26 against symbol `memcpy@@LIBC' defined in .text section in /Users/worker/tasks/task_1558764312/magicleap/v0.17.0/lumin/usr/lib/libc.so
/Volumes/android/aosp_sdk_osx/repos/external/libcxx/../../external/libcxxabi/../libcxx/include/string:655:(.text._ZN10__cxxabiv112_GLOBAL__N_120parse_template_paramINS0_2DbEEEPKcS4_S4_RT_+0x6b0): relocation truncated to fit: R_AARCH64_CALL26 against symbol `memcpy@@LIBC' defined in .text section in /Users/worker/tasks/task_1558764312/magicleap/v0.17.0/lumin/usr/lib/libc.so
/Volumes/android/aosp_sdk_osx/repos/external/libcxx/../../external/libcxxabi/../libcxx/include/string:655:(.text._ZN10__cxxabiv112_GLOBAL__N_120parse_template_paramINS0_2DbEEEPKcS4_S4_RT_+0x75c): additional relocation overflows omitted from the output
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [/Users/worker/tasks/task_1558764312/repo/target/magicleap/aarch64-linux-android/debug/debug_lumin_clang-3.8_aarch64/Servo2D.mk:23: /Users/worker/tasks/task_1558764312/repo/target/magicleap/aarch64-linux-android/debug/debug_lumin_clang-3.8_aarch64/Servo2D] Error 1
Packaging Magic Leap exited with return value 1
```

--- a/etc/taskcluster/decision_task.py
+++ b/etc/taskcluster/decision_task.py
@@ -815,8 +815,8 @@ def magicleap_build_task(name, build_type):
         macos_build_task(name)
         .with_treeherder("MagicLeap aarch64", build_type)
         .with_directory_mount(
-            "https://servo-deps.s3.amazonaws.com/magicleap/macos-sdk-v0.17.0.tar.gz",
-            sha256="e81de47ad963891ac68768d93ab5a36ed3af3a3efebb4dbc4db2e65647d57655",
+            "https://servo-deps.s3.amazonaws.com/magicleap/macos-sdk-v0.20.0%2Bndk19c.tar.gz",
+            sha256="d5890cc7612694d79e60247a5d5fe4d8bdeb797095f098d56f3069be33426781",
             path="magicleap"
         )
         .with_directory_mount(
@@ -826,7 +826,7 @@ def magicleap_build_task(name, build_type):
         )
         # Early script in order to run with the initial $PWD
         .with_early_script("""
-            export MAGICLEAP_SDK="$PWD/magicleap/v0.17.0"
+            export MAGICLEAP_SDK="$PWD/magicleap/v0.20.0+ndk19c"
             export MLCERT="$PWD/magicleap/TempSharedCert.cert"
         """)
         .with_script("""

